src/client/redux/auth/operations.js

```diff
@@ -55,7 +59,7 @@ const refresh = createAsyncThunk('auth/refresh', async (_, thunkAPI) => {
 
   try {
     setAuthHeader(persistedToken);
-    const response = await axios.get('/current');
+    const response = await axios.post('/current');
```

Comment:
🦸‍♂️

Reply:
I had to change to post because on backend post is used for current endpoint instead of get.